app.py

```diff
@@ -0,0 +1,38 @@
+import streamlit as st
+import importlib
+
+st.set_page_config(layout="wide")
```

Comment:
Just to make the site look a bit more professional:

```
st.set_page_config(
    page_title="LINC Dashboard",   
    page_icon="🧠",                  
    layout="wide", 
    initial_sidebar_state="expanded"
)
```

You could use a `favicon.ico` as well if you want https://github.com/lincbrain/linc-archive/blob/master/web/public/favicon.ico instead of the brain emoji

Reply:
Good idea.  I have added this.